dist/licenses/LICENSE-prometheus-common.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Remove this please.

Reply:
Done, remove this line.

Reply:
> Done, remove this line.

Not remove this line, remove this file

Reply:
Is this file deleted because Apache 2.0 does not need to declare a license file?

Reply:
> Is this file deleted because Apache 2.0 does not need to declare a license file?

See my comment https://github.com/apache/skywalking-satellite/pull/42#discussion_r642734047

Because it's standard Apache 2.0 license, we don't need to copy it again because we already have it in our root directory

Reply:
Done, thanks.